en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-apis-as-immutable-gateway.md

```diff
@@ -0,0 +1,63 @@
+# Deploy APIs as immutable gateway
+
+In this case, the APIs will be deployed during the startup. After startup, no APIs can be added using apictl. 
+For that, the apictl projects needs to be mounted to the `docker-compose/resources/adapter/artifacts/apis` directory 
+(default location) of the adapter. 
+
```

Comment:
Deploy APIs as an Immutable gateway?

Reply:
In this case, the APIs can be deployed to the gateway in an immutable manner. For that, the apictl projects need to be mounted to the `docker-compose/resources/adapter/artifacts/apis` directory 
(default location) of the adapter. At the startup of the adapter, it will pick the projects. After startup, APIs should be added using apictl.

Reply:
fixed with https://github.com/wso2/docs-apim/pull/4968/commits/58c025c764f9d7349b1e12c3709188929873dce5